types/peer.js

```
@@ -1648,47 +1652,51 @@ class Peer extends Service {
-        this._handleSessionOfferGenericMessage(message, origin, socket, signerPubkeyHex);
+        this._handleSessionOfferGenericMessage(msg, origin, socket, signerPubkeyHex);
         break;
       case 'P2P_SESSION_OPEN':
-        this._handleSessionOpenGenericMessage(message, origin, signerPubkeyHex);
+        this._handleSessionOpenGenericMessage(msg, origin, signerPubkeyHex);
         break;
       case 'P2P_CHAT_MESSAGE':
-        this.emit('chat', message);
-        const relay = Message.fromVector(['ChatMessage', JSON.stringify(message)]);
+        this.emit('chat', msg);
+        const relay = Message.fromVector(['ChatMessage', JSON.stringify(msg)]);
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)